src/qibolab/platforms/abstract.py

```diff
@@ -174,16 +175,17 @@ def reload_settings(self):
                 readout, drive, flux = settings["qubit_channel_map"][q]
                 feedback = None
             else:
-                readout, drive, flux, feedback = settings["qubit_channel_map"][q]
+                readout, drive, flux, feedback, twpa = settings["qubit_channel_map"][q]
```

Comment:
This will probably break other platforms where `settings["qubit_channel_map"]` has different length (not 3 or 5). In general this block of code was just a temporary fix to make it work with refactor 1 and I don't like it much. 

I would prefer if `settings["qubit_channel_map"]` has fixed dimensions and is populated with `None` in cases that are not relevant (such as flux if the qubit is not flux tunable).

Reply:
Indeed I'm playing around with this code to make it compatible with `tii1q_b1` and it broke the platform.